layouts/partner/aws.html

```diff
@@ -251,82 +157,35 @@ <h3>Programmatic Infrastructure as Code</h3>
-                </p>
-
-                <h5>Application and Infrastructure Together</h5>
-                <p>
-                    Crosswalk enables you to blur the lines between application and infrastructure code enabling you to author an entire full-stack application in one program. With
-                    support for inline Lambda functions and ease-of-use helper functions, building robust applications on AWS has never been easier.
-                </p>
-                {{ partial "sparkle-arrow-cta" (dict "url" "/docs/iac/clouds/aws/guides/" "text" "Learn More About Crosswalk" "textColor" "grey" "textAlign" "left") }}
-            </div>
-        </div>
-    </section>
-
-    <!-- Start AWS re:Invent takeover content -->
-    <section id="workshops" class="container mx-auto text-center mb-12 mt-24">
-        <h2 class="mb-16">Watch a workshop</h2>
-        <ul class="flex flex-col px-0 lg:flex-row list-none text-left items-center">
-            {{ $webinars := (where $.Site.Pages "Type" "webinars") }}
-
-            {{ $reInventWorkshops := slice }}
-            {{ range $webinars }}
-                {{ if eq .Params.title "Building a Kubernetes Platform in Amazon EKS with Pulumi" }}
-                    {{ $reInventWorkshops = $reInventWorkshops | append . }}
-                {{ else if eq .Params.title "Deploying Microservices with Pulumi and AWS Lambda" }}
-                    {{ $reInventWorkshops = $reInventWorkshops | append . }}
-                {{ else if eq .Params.url_slug "introduction-to-pulumi" }}
-                    {{ $reInventWorkshops = $reInventWorkshops | append . }}
-                {{ end }}
-            {{ end }}
-
-            {{ range $reInventWorkshops }}
-                {{ $link := printf "/resources/%s" .Params.url_slug }}
-
-                {{ $displayDate := "" }}
-                {{ if .Params.multiple }}
-                    {{ $displayDate = dateFormat "January 2, 2006" (index (.Params.multiple) 0).datetime }}
-                {{ else }}
-                    {{ $displayDate = dateFormat "January 2, 2006" .Params.main.sortable_date }}
-                {{ end }}
-
-                {{ $tileOptions := (dict "pageContext" $pageContext "external" false "filters" "" "link" $link "icon" "users" "description" .Params.meta_desc "displayDate" $displayDate "data" . ) }}
-                {{ partial "content-tile.html" $tileOptions }}
+            {{ with .Params.crosswalk }}
+                <div class="w-full order-first lg:order-last lg:w-1/2 px-8">
+                    <h3>{{ .header }}</h3>
+                    <p class="mt-0">{{ .body | markdownify }}</p>
+                    {{ range .points }}
+                        <h5>{{ .header }}</h5>
+                        <p>{{ .body | markdownify }}</p>
+                    {{ end }}
+                    {{ partial "sparkle-arrow-cta" (dict "url" .link "text" .cta "textColor" "grey" "textAlign" "left") }}
```

Comment:
Same here. The code block needs some white space above it to balance out the white space that exists below